my-project/models/api_message.py

```diff
@@ -1,20 +1,20 @@
-import dataclasses
 from datetime import datetime
 import time
 import uuid
 
-import google.cloud.firestore as g_firestore
-
 from modules.nanoseconds import datetime_with_nanoseconds
 
 
-@dataclasses.dataclass
-class ApiMessages():
-    message: str
-    
-    def set_message(self, collection: g_firestore):
-        collection.document().set({
-            'id': str(uuid.uuid4()),
-            'message': self.message,
-            'timestamp': datetime_with_nanoseconds(datetime.fromtimestamp(round(time.time())))
-        })
\ No newline at end of file
+_id = "id"
+_message = "message"
+_timestamp = "timestamp"
+
+
+def set_message(message: str) -> dict:
+    return {
+        _id: str(uuid.uuid4()),
+        _message: message,
+        _timestamp: datetime_with_nanoseconds(
+            datetime.fromtimestamp(round(time.time()))
+        ),
+    }
```

Comment:
```python
{
  "id: str(uuid.uuid4()),
}
```

Reply:
```python
@dataclasses.dataclass
class Message:
  id: str
  content: str
  timestamp: datatime.datetime
  
   def __init__(self, message):
        id = str(uuid.uuid4())
        content = message
        timestamp = datetime_with_nanoseconds(
            datetime.fromtimestamp(round(time.time()))
        )

message = Message("こんにちは")
message.__dict__
```

Reply:
`timestamp = datetime_with_nanoseconds(
            datetime.fromtimestamp(round(time.time()))
        )`

initにdatetimeを入れて固定されないかを確認します

Reply:
前回コーディング時に値が固定されてしまったのは
クラス変数の初期値を設定して初期化設定(__init__)していなかったためでした！
https://zenn.dev/enven/articles/8b80ff38461b4ff329aa
```suggestion
@dataclasses.dataclass
class Message:
  id: str
  content: str
  timestamp: datatime.datetime
  
   def __init__(self, message):
        self.id = str(uuid.uuid4())
        self.content = message
        self.timestamp = datetime_with_nanoseconds(
            datetime.fromtimestamp(round(time.time()))
        )
```
使わせていただきます！

Reply:
<img width="708" alt="スクリーンショット 2022-06-17 19 01 15" src="https://user-images.githubusercontent.com/50030974/174278442-2bc36d5c-d5d2-4fd2-bf2a-061a76f35952.png">